src/main/scala/EnsimePlugin.scala

```diff
@@ -427,12 +427,12 @@ object EnsimePlugin extends AutoPlugin {
     def jarSrcsFor(config: Configuration) = updateClassifiersReport.select(
       configuration = configFilter(config),
       artifact = artifactFilter(classifier = Artifact.SourceClassifier)
-    ).toSet ++ (ensimeUnmanagedSourceArchives in config in projectRef).gimme
```

Comment:
hmm, I thought we got rid of the command support stuff when you converted us to Task, how come `gimme` was still around?

Reply:
`gimme` was still used in the tasks, I remember its about using a dynamic configuration when getting a setting value.

Reply:
I think it was always supposed to be a hack to access settings / tasks from commands :-D it appears to have been repurposed

Reply:
Yeah, in `projectData` methods like `sourcesFor` cannot accept a `Configuration` as a parameter if we were to use the task macro. So I might have reused `gimme`.